part-spec/definitions.json

```diff
@@ -479,63 +479,87 @@
           "type": "number"
         }
       }
-    },    
+    },
     "package": {
       "type": "object",
       "properties": {
         "length": {
           "description": "length of a side of a package",
-          "type": "number"
+          "$comment": "units of millimeter",
+          "$ref": "#/$definitions/unit"
         },
         "width": {
           "description": "width of a side of a package",
-          "type": "number"
+          "$comment": "units of millimeter",
+          "$ref": "#/$definitions/unit"
         },
         "height": {
           "description": "height of a package",
-          "type": "number"
+          "$comment": "units of millimeter",
+          "$ref": "#/$definitions/unit"
         },
-        "dimensionUnit": {
-          "description": "unit used to describe package dimensions",
-          "enum": [
-            "mils",
-            "millimeter"
-          ]
+        "nominalFootprints": {
+          "description": "references to external footprints in standard CAD formats",
+          "type": "array",
+          "items": {
+            "$ref": "#/$definitions/externalFile"
+          }
         },
-        "standardPackageSize": {
-          "description": "name of standard package size (imperial)",
-          "enum": [
-            "0201",
-            "0402",
-            "0603",
-            "0805",
-            "1206"
-          ]
+        "breakoutExamples": {
+          "description": "references to external board file that contains layout breakout example",
+          "type": "array",
+          "items": {
+            "$ref": "#/$definitions/externalFile"
+          }
         },
-        "standardPackageType": {
-          "description": "name of standard package types",
-          "example": [
-            "bga",
-            "wcsp",
-            "qfn",
-            "tssop"
-          ]
+        "partModelInformation": {
+          "description": "reference to an external XML file that contains a part model in IPC/DAC2552 format",
+          "$ref": "#/$definitions/externalFile"
         }
       }
     },
-    "ratio":{
-    "type": "object",
-    "properties": {
-      "numerator":{
-        "description": "numerator of ratio",
-        "type": "number"
-      },
-      "denominator":{ 
-        "description": "denominator of ratio",
-        "type": "number"
+    "externalFile": {
+      "type": "object",
+      "properties": {
+        "cadSoftware": {
+          "description": "name of CAD software file is written in, if applicable",
+          "type": "string"
+        },
+        "standardReferenced": {
+          "description": "name of the standard the file is written in, if applicable",
+          "type": "string"
+        },
+        "fileExtension": {
+          "description": "type of file referenced by link",
+          "examples": [
+            "brd",
+            "mod"
+          ],
+          "type": "string"
+        },
+        "filePath": {
```

Comment:
Do we want fileName instead or filePath? I'm thinking about cases where the vendors can have different folders based on EDA tools supported. It may be more challenging to list a path.